client/logic/globals/types.gd

```diff
@@ -62,6 +68,69 @@ func _generate_types() -> void:
 	secondary_damage = _get_damage_values(path, unit_types)
 
 
+func add_text_to_gdscript(
+	file: FileAccess, name: String, text: String, context: String = ""
+) -> void:
+	var sanitized_name: String = sanitizer_regex.sub(name, "_")
+	file.store_string(
+		(
+			'var _%s: String = tr("%s", "%s")\n'
+			% [sanitized_name, text.replace('"', '\\"'), context.replace('"', '\\"')]
+		)
+	)
+
+
+func _generate_labels() -> void:
+	var file_dumper: FileAccess = FileAccess.open(
+		"res://assets/translations/dynamic_string_loader.gd", FileAccess.WRITE
+	)
+	if file_dumper.get_error():
+		print("Error opening dynamic string loader file")
+
+	file_dumper.store_string("# this file is autogenerated in types.tscn,\n")
+	file_dumper.store_string("# you can update opening it in godot\n")
```

Comment:
What do you think of this enhanced comment?
`This file is automatically generated from the types.tscn scene. \n`
`To modify the generated code, please edit the types.tscn scene directly in Godot. \n`